tests/unit/msgbuff.c

```diff
@@ -9,12 +9,28 @@
 #include "test-common.h"
 #include "nccl_ofi_msgbuff.h"
 
+static inline uint16_t subtract(const uint16_t size, const uint16_t minuend, const uint16_t subtrahend)
```

Comment:
The msgbuff unit test commit should be part of the commit that changes the msgbuff interface.

Also, the subtract function irritates me for the same reason called out in the previous commit, and let's figure out how not to have it exist in two places.